packages/backend/src/peripherals/database/AggregatedReportRepository.ts

```diff
@@ -21,24 +22,27 @@ export class AggregatedReportRepository extends BaseRepository {
   async getDaily(): Promise<AggregatedReportRecord[]> {
```

Comment:
tbh I would prefer if we pass the ValueType there instead of hardcoding it, but if you plan to work on it in the next PR it is okay to me